packages/react-app/src/App.jsx

```diff
@@ -107,10 +113,10 @@ function App(props) {
   };
 
   /* 💵 This hook will get the price of ETH from 🦄 Uniswap: */
-  const price = useExchangeEthPrice(targetNetwork, mainnetProvider);
+  const price = useExchangeEthPrice(targetNetwork, mainnetProvider, 1000);
```

Comment:
Is ok 1000 here? Maybe MAINNET_RPC_POLL_TIME?

Reply:
Fixed, thanks! ;)